### ISSUE TITLE
multiple initial seed URLs

### ISSUE THREAD

**Issue by zyy20853** [crawljax/crawljax#501]
Dear developers, I have multiple initial URLs to crawl. Instead of following deeper into each of them, I only need the page content. Now what I did is to close the context in the OnNewState Plugin, and restart a new builder of a new URL in the main loop. I think this is a common situation, but I can't find any info on the web. Is there any clever way to do this?  It waste too much time every time configuring a new builder.

my code is something like:

while(URL list is not empty) {
    CrawljaxConfigurationBuilder builder = CrawljaxConfiguration
					.builderFor( a new URL);
    CrawljaxRunner crawljax = new CrawljaxRunner(builder.build());
    crawljax.call();
}

onNewState(CrawlerContext context, StateVertex newState) {
    context.stop();
    // do some computation here
}

Thanks.

**Comment by zyy20853** [comment]
My situation is a little similar to issue #93 , did you solve that one?
